bin/utils.js

```diff
@@ -16,3 +17,12 @@ function parseFilePaths(argv) {
 }
 
 module.exports = { parseFilePaths, parseOptions };
+
+function renamePrompt(props) {
+  if (!props?.length) return [];
+  const prompt = require('prompt-sync')({ sigint: true });
```

Comment:
Why is it required here?

Reply:
Because I wanted to scope it inside of that function. Since the prompt is only used inside of this function, my thinking was that I want the garbage collector to destroy the prompt after it has been used.